ui/src/lib/features/k8s/store.ts

```diff
@@ -191,7 +193,7 @@ export class ResourceStore<T extends KubernetesObject, U extends CommonRow> impl
     this.#eventSource.onmessage = ({ data }) => {
       try {
         this.#table = this.#tableCallback(JSON.parse(data))
-        this.#resources.set(this.#table)
+        this.#resources.set([])
```

Comment:
```suggestion
        this.#resources.set(this.#table)
```
assume you did this for debugging and just forgot to put back